index.html

```diff
@@ -1323,23 +1336,22 @@ <h3 id="name-request-parameters">
-<p id="section-2.3-3">Unsigned plaintext Request Objects, containing <code>none</code> in the <code>alg</code> value of the JOSE header MUST not be supported.<a href="#section-2.3-3" class="pilcrow">¶</a></p>
-<p id="section-2.3-4">The Request Object MAY also be encrypted using <a href="https://tools.ietf.org/html/rfc7516">JWE</a>, however the inner payload MUST be a valid <a href="https://tools.ietf.org/html/rfc7519">JWT</a> signed by the Client who created the request.<a href="#section-2.3-4" class="pilcrow">¶</a></p>
-<p id="section-2.3-5">Public private key pairs are used by a requesting Client to establish a means of binding to the resulting credential. A Client making a credential request to an OpenID Provider must prove control over this binding mechanism during the request, this is accomplished through the use of a <a href="https://openid.net/specs/openid-connect-core-1_0.html#SignedRequestObject">signed request</a> defined in OpenID Connect Core.<a href="#section-2.3-5" class="pilcrow">¶</a></p>
-<p id="section-2.3-6">To bind the credential request to the Client making the request, the Request Object MUST be signed by the Client using a public private key pair the Client is in possession of.<a href="#section-2.3-6" class="pilcrow">¶</a></p>
-<p id="section-2.3-7">If the Request Object signing validation fails or is missing, the OpenID Connect Provider MUST respond to the request with the Error Response parameter, <a href="https://openid.net/specs/openid-connect-core-1_0.html#AuthError">section 3.1.2.6.</a> with Error code: <code>invalid_request_object</code>.<a href="#section-2.3-7" class="pilcrow">¶</a></p>
+<p id="section-2.3-1">Public private key pairs are used by a requesting Client to establish a means of binding to the resulting credential. A Client making a credential request to an OpenID Provider must prove control over this binding mechanism during the request, this is accomplished through the modified usage of a <a href="https://openid.net/specs/openid-connect-core-1_0.html#SignedRequestObject">signed request</a> defined in OpenID Connect Core.<a href="#section-2.3-1" class="pilcrow">¶</a></p>
+<p id="section-2.3-2">Usage of the <code>request</code> parameter as defined in section <a href="https://openid.net/specs/openid-connect-core-1_0.html#ClaimsParameter">5.5</a> of OpenID Connect core is REQUIRED in a credential request.<a href="#section-2.3-2" class="pilcrow">¶</a></p>
+<p id="section-2.3-3">The value of the <code>request</code> parameter MUST either be a valid <a href="https://tools.ietf.org/html/rfc7519">JWT</a> or <a href="https://tools.ietf.org/html/rfc7516">JWE</a> whose claims are the credential request parameters, however the inner payload MUST be a valid <a href="https://tools.ietf.org/html/rfc7519">JWT</a> signed by the Client who created the request.<a href="#section-2.3-3" class="pilcrow">¶</a></p>
```

Comment:
nested JWT rather than specifying JWE here?

Reply:
Just copied this section from signed request object so might leave as is for now, but agree the concept of nested JWT would be a useful concept to introduce